internal/drive/drive.go

```diff
@@ -5,21 +5,39 @@ import (
 	"io"
 )
 
+// Metadata contains name, size (in bytes), and content hash of a remote file
+type Metadata struct {
+	Name string
+	Size uint64
+	Hash string
+}
```

Comment:
@paddlesteamer Why make this public?

Reply:
Used in here: https://github.com/paddlesteamer/cloudstash/blob/033e404c2a2ddefd320e1d660f6bc7e32d33692d/internal/manager/background.go#L30